### PR TITLE
Update UT for BuildCommandTest.cs

### PR DESCRIPTION
**Pull request by qinezh**
@vwxyzh @chenkennt @ansyral @hellosnow @superyyrrzz

### CODE REVIEW

**Comment by ansyral** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:+1:

**Comment by superyyrrzz** [comment]
:shipit:

File: test/docfx.Tests/BuildCommandTest.cs [hunk 1]
```diff
@@ -237,10 +266,10 @@ public void TestParseCommandOption()
...
             }).Config;
-            Assert.Equal(config3.GlobalMetadata["key"], "--globalMetadata");
-            Assert.Equal(config3.GlobalMetadata["global1"], "1");
-            Assert.Equal(config3.GlobalMetadata["global2"], "2");
-            Assert.Equal(config3.GlobalMetadata["global.deprecated"], "deprecated");
+            Assert.Equal("--globalMetadata", config3.GlobalMetadata["key"]);
+            Assert.Equal("1", config3.GlobalMetadata["global1"]);
+            Assert.Equal("2", config3.GlobalMetadata["global2"]);
+            Assert.Equal("deprecated", config3.GlobalMetadata["global.deprecated"]);
         }
```
Comment (superyyrrzz) [inline test/docfx.Tests/BuildCommandTest.cs]: Minor: Split into several UT and use better function name?

Comment (qinezh) [inline test/docfx.Tests/BuildCommandTest.cs]: good idea

**Pull request merged**